_includes/api-selector.html

```diff
@@ -28,6 +28,7 @@
                     <option value="trading_times">Trading Times</option>
                     <option value="verify_email">Verify an email address</option>
                     <option value="website_status">Website status</option>
+                    <option value="trading_durations">Trading Durations</option>
```

Comment:
@yngshan Inconsistent casing - please look at the other entries: we should decide on a standard format and stick to it. I'd recommend lowercase, there's nothing special about durations here:

```suggestion
                    <option value="trading_durations">Trading durations</option>
```

but please also apply this to the other items.